tests/typing.py

```diff
@@ -33,3 +36,6 @@ class MockHAClientWebSocket(ClientWebSocketResponse):
 type RecorderInstanceGenerator = Callable[..., Coroutine[Any, Any, Recorder]]
 """Instance generator for `homeassistant.components.recorder.Recorder`."""
 type WebSocketGenerator = Callable[..., Coroutine[Any, Any, MockHAClientWebSocket]]
+
+type Patcher = _patch_default_new
```

Comment:
Why should it be an alias for `_patcher_default_new` and not `_patcher`?

Reply:
`_patcher` is also a callable, same as `patch`
However, this made me realise that I could use `_patch` directly, and since it's available at runtime I don't need the alias